service/src/test/java/bio/terra/externalcreds/service/LinkedAccountServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package bio.terra.externalcreds.service;
+
+import static org.mockito.Mockito.when;
+
+import bio.terra.externalcreds.BaseTest;
+import bio.terra.externalcreds.dataAccess.LinkedAccountDAO;
+import bio.terra.externalcreds.models.LinkedAccount;
+import bio.terra.externalcreds.services.LinkedAccountService;
+import java.sql.Timestamp;
+import java.util.UUID;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+public class LinkedAccountServiceTest extends BaseTest {
+  @Autowired private LinkedAccountService linkedAccountService;
+
+  @MockBean private LinkedAccountDAO linkedAccountDAO;
```

Comment:
I don't think we should use a mock here, use the real thing. In this particular test it does not really matter but future tests may rely on complex database state making for a complicated and brittle mock.